nidmviewerfsl/lib/statFormat.py

```diff
@@ -178,6 +178,9 @@ def contrastVec(data, v_min, v_max):
     # This import is needed only in this function.
     from matplotlib import pyplot as plt
 
+    # This import is needed only in this function.
+    from matplotlib import pyplot as plt
```

Comment:
The matplotlib import is done twice? Here and line 178?

Question: is it good practice to add an import inside a function if only required by that function?

Reply:
Ah yes! Sorry this kept moving about during the rebase! 

To be honest... the main reason I did this was because the PEP8 bot complains if I put it near the top as the function on line 17 must be called first (`matplotlib.use('Agg')`) but the PEP8 bot wants all imports before any other line.

Having looked online though I think there are good arguments for doing this either way: 

https://stackoverflow.com/questions/9037733/python-importing-at-function-level-vs-module-level
https://stackoverflow.com/questions/9614051/import-at-module-level-or-at-function-level

I think it also makes sense here as this function creates the contrast vector images and may not be run if the user is trying to display FSL data, making the import unnecessary.